### PR TITLE
fix(ghost-button): use theme tokens

### PR DESCRIPTION
**Pull request by SimonFinney**
#### Changelog

**Changed**

- Uses theme token for ghost button active and hover.

### CODE REVIEW

**Comment by emyarod** [comment]
hi @SimonFinney thanks for tackling this! is there an issue/ticket associated with this PR?

File: packages/components/src/components/button/_button.scss [hunk 1]
```diff
@@ -132,10 +132,10 @@
 
     &:hover,
     &:active {
-      color: $ibm-color__blue-70;
+      color: $interactive-04;
 
       .#{$prefix}--btn__icon path {
```
Comment (joshblack) [inline packages/components/src/components/button/_button.scss:137]: This is unrelated to the PR but would we need to open up this selector for other types of svg nodes like `circle`/`rect`/etc?

**Review by joshblack — APPROVED** [review]

File: packages/components/src/components/button/_button.scss (hunk 1 above)
Comment (asudoh) [reply packages/components/src/components/button/_button.scss:137]: CC @emyarod for his thoughts.

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @SimonFinney!

File: packages/components/src/components/button/_button.scss (hunk 1 above)
Comment (emyarod) [reply packages/components/src/components/button/_button.scss:137]: yes we can target `svg` instead

**Pull request merged**